develop/start/config/browserSwitch.js

```diff
@@ -18,14 +18,14 @@ module.exports = function browserSwitch(projectDir, browserVendor) {
           port: 8081
         }).apply(compiler)
 
-        // new RunEdgeExtension({
-        //   extensionPath: projectDir,
-        //   port: 8082
-        // }).apply(compiler)
+        new RunEdgeExtension({
+          extensionPath: projectDir,
+          port: 8082
+        }).apply(compiler)
       }
     }
   }
 
-  // TODO: Should default to user browser, not Chrome
-  return new RunChromeExtension({extensionPath: projectDir})
+  // Default to user browser
+  return browserVendor
```

Comment:
can you return Chrome here? at this point I can't support all browsers

```suggestion
  // TODO: Should default to user browser, not Chrome
  return new RunChromeExtension({extensionPath: projectDir})
```